### PR TITLE
fix: updating logs behavior

### PR DESCRIPTION
**Pull request by tonysnowboardunderthebridge**

### CODE REVIEW

File: src/renderer/src/components/application/application-logs/Log.jsx [hunk 1]
```diff
@@ -1,19 +1,21 @@
 'use strict'
 import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
-import commonStyles from '~/styles/CommonStyles.module.css'
+// import commonStyles from '~/styles/CommonStyles.module.css'
```
Comment (marcopiraccini) [inline src/renderer/src/components/application/application-logs/Log.jsx:4]: Delete commented import

**Comment by tonysnowboardunderthebridge** [comment]
Hi @marcopiraccini @mcollina 

this loom explain the behavior of the logs

https://www.loom.com/share/9b3e3d26a2df40c18df1f271481319ce

File: src/renderer/src/components/application/application-logs/Log.jsx (hunk 1 above)
Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/application/application-logs/Log.jsx]: Done!

**Review by marcopiraccini — APPROVED** [review]
lgtm

**Pull request merged**